web/js/edsc.js

```diff
@@ -1,9 +1,14 @@
+import edscArc from './edsc/arc';
```

Comment:
What's the value of the `edsc` prefix? Didn't we agree that `edsc` isn't really relevant to this code anymore?